src/main/scala/com/fulcrumgenomics/sopt/cmdline/ClpArgumentDefinitionPrinting.scala

```diff
@@ -42,6 +42,15 @@ object ClpArgumentDefinitionPrinting {
   private[cmdline] val ArgumentDefaultValuePrefix: String = "Default:"
   private[cmdline] val ArgumentOptionalValue: String = "Optional"
 
+  /** A collection of non-printing ASCII characters and their string literals. */
+  private[cmdline] val NonPrintingCharacters: Map[String, String] = Map(
```

Comment:
Maybe this should be `NonPrintingEscapedCharacters` since we don't need to escape spaces.